firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandlerAutomated.cs

```diff
@@ -168,6 +170,7 @@ public class UIHandlerAutomated : UIHandler {
         TestTerminateMultiDBIndependently,
         TestTerminateAppWithMultiDB,
         TestRestartCustomFirestore,
+        TestAndOrQueriesCompositeIndexesRequired
```

Comment:
It looks like `TestAndOrQueriesCompositeIndexesRequired` tests accidentally got deleted?